integration-cli/docker_api_attach_test.go

```diff
@@ -85,8 +87,10 @@ func (s *DockerSuite) TestGetContainersWsAttachContainerNotFound(c *check.C) {
 	status, body, err := sockRequest("GET", "/containers/doesnotexist/attach/ws", nil)
 	c.Assert(status, checker.Equals, http.StatusNotFound)
 	c.Assert(err, checker.IsNil)
-	expected := "No such container: doesnotexist\n"
-	c.Assert(string(body), checker.Contains, expected)
+	var resp types.ErrorResponse
+	c.Assert(json.Unmarshal(body, &resp), checker.IsNil)
```

Comment:
That's a nit and not required for the PR (so it's more a *quick note*). We might want to add a helper for that (`checkError(…)`) :angel: